syntax = "proto3";

service MetadataService {

    // Account Manager
    rpc newAccount(NewAccountRequest) returns (NewAccountResponse) {}
}

message NewAccountRequest {
    string username = 1;
    string password = 2;
}

message NewAccountResponse {
    string username = 1;
}